host/xtest/regression_1000.c

```diff
@@ -834,13 +875,27 @@ static void xtest_tee_test_1009(ADBG_Case_t *c)
 
 static void xtest_tee_test_1010(ADBG_Case_t *c)
 {
-	unsigned n;
+	uint32_t n;
```

Comment:
We can't seem to get enough of unsigned types. ;-)
I would have used only `size_t` here if nothing else just to avoid the distraction.
For instance, why is `uint32_t` used for `n` instead of `unsigned int` which is used for `idx`?

Reply:
I used the native type used in the TA API: `uint32_t` for the operation command. But true that it's a bit silly. I'll move former to `unsigned int`. Sorry bothering.